.storybook/test-storybook-canvas.config.js

```diff
@@ -1,5 +1,8 @@
 const DEFAULT_SCENARIO_CONFIG = {
     readyEvent: "GDC_GEO_CANVAS_READY",
+    // Color/Position of labels in Mapbox is rendered incorrectly sometimes
+    // Wait for 200 milliseconds to fix it
+    delay: 200,
```

Comment:
Should this config aim only to specific case, like `GeoPushpinChart example`, rather than all cases?

Reply:
Hi @naibinh,
I was wondering if, the stories are working well now, may stop working in the future.
Who know?

Reply:
So we can  apply it later, right?
Why I'm asking is it may slow `extended test - storybook` a bit, not only to current test cases but also future ones.

Reply:
updated